README.md

```diff
@@ -93,28 +93,24 @@ You can build RIE into a base image. Download the RIE from GitHub to your local
 3. Install the emulator package and change `ENTRYPOINT` to run the new script by adding the following lines to your Dockerfile:
 
     To use the default x86\_64 architecture
-
```

Comment:
This PR is also changing the formatting, it is inconsistently applied as well. The change needs updated to not affect the spacing.

Reply:
Good point. Updated.

Reply:
Thanks!